Books-Labels/schema.sql

```diff
@@ -0,0 +1,24 @@
+CREATE DATABASE catalog;
```

Comment:
I think this command is redundant because if we wanted to run all the commands in the file at once we would create a database called `catalog` but the rest of the tables would be created in another database.